components/Person/PersonDetailForm.js

```diff
@@ -567,8 +581,8 @@ PersonDetailForm.propTypes = {
     twitter: PropTypes.string,
     website: PropTypes.string,
     pronoun: PropTypes.object,
-    imgUrl: PropTypes.string,
-    imgUrlSm: PropTypes.string,
+    job: PropTypes.string,
+    imgUrl: PropTypes.any,
```

Comment:
it seems 
 imgUrlSm: PropTypes.string,
has been removed - I think this should still be included

Reply:
Ive added back in now ! Thanks heaps for the feedback